### PR TITLE
Or 1084 titond backend add data transport layer to resource api

### PR DESCRIPTION
**Pull request by ohbyeongmin**
Add data transport layer api

### CODE REVIEW

File: testdata/data-transport-layer/statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          envFrom:
+            - configMapRef:
+                name: data-transport-layer
+          ports:
+            - containerPort: 7878
+              protocol: TCP
+          volumeMounts:
+            - name: data-transport-layer-volume
+              mountPath: /db
+              subPath: data-transaport-layer
```
Comment (rlgns98kr) [inline testdata/data-transport-layer/statefulset.yaml:32]: We have to edit this to use other host path of efs for each component.

Comment (ohbyeongmin) [reply testdata/data-transport-layer/statefulset.yaml:32]: Yes! I'll be implementing it while testing in the next ticket.
Thanks for the reminder! 🙇‍♂️

File: testdata/data-transport-layer/configMap.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+kind: ConfigMap
+metadata:
+  name: data-transport-layer
+data:
+  DATA_TRANSPORT_LAYER__ETH_NETWORK_NAME: hardhat
+  DATA_TRANSPORT_LAYER__NODE_PACKAGE_VERSION: "0.5.47"
+  DATA_TRANSPORT_LAYER__L1_START_HEIGHT: "1"
+  DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT: "http://l1chain-svc:8545"
+  DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT: "http://l2geth-svc:8545"
+  DATA_TRANSPORT_LAYER__L2_CHAIN_ID: "17"
```
Comment (rlgns98kr) [inline testdata/data-transport-layer/configMap.yaml:11]: Okay, we have to give each chain_id to user for each network. I think we can fix it later!

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 😍

File: testdata/data-transport-layer/configMap.yaml (hunk 2 above)
Comment (ohbyeongmin) [reply testdata/data-transport-layer/configMap.yaml:11]: thank you!

**Pull request merged**